src/tests/Microsoft.PowerFx.Interpreter.Tests/FileExpressionEvaluationTests.cs

```diff
@@ -39,7 +39,8 @@ public void Canvas_Float(ExpressionTestCase testCase)
             var features = new Features()
             {
                 TableSyntaxDoesntWrapRecords = true,
-                ConsistentOneColumnTableResult = true
+                ConsistentOneColumnTableResult = true,
+                StronglyTypedBuiltinEnums = true,

```

Comment:
Canvas does not have StronglyTypedBuiltInEnums by default (or at all) at the moment, is this right?

Reply:
yes, will remove this.